packages/integration-tests/src/test-e2e/iam_access_drift.test.ts

```diff
@@ -0,0 +1,348 @@
+import { after, before, describe, it } from 'node:test';
+import { execa } from 'execa';
+import path from 'path';
+import { TestBranch, amplifyAppPool } from '../amplify_app_pool.js';
+import { BackendIdentifier } from '@aws-amplify/plugin-types';
+import {
+  AccessAnalyzerClient,
+  CheckNoNewAccessCommand,
+  CheckNoNewAccessCommandOutput,
+} from '@aws-sdk/client-accessanalyzer';
+import {
+  CloudFormationClient,
+  DeleteStackCommand,
+} from '@aws-sdk/client-cloudformation';
+import {
+  GetPolicyCommand,
+  GetPolicyCommandOutput,
+  GetPolicyVersionCommand,
+  GetPolicyVersionCommandOutput,
+  GetRoleCommand,
+  GetRoleCommandOutput,
+  GetRolePolicyCommand,
+  GetRolePolicyCommandOutput,
+  IAMClient,
+  ListRolePoliciesCommand,
+  ListRolePoliciesCommandOutput,
+} from '@aws-sdk/client-iam';
+import fsp from 'fs/promises';
+import { e2eToolingClientConfig } from '../e2e_tooling_client_config.js';
+import { DeployedResourcesFinder } from '../find_deployed_resource.js';
+import { NpmProxyController } from '../npm_proxy_controller.js';
+import { AccessCheckPolicyType } from '@aws-sdk/client-accessanalyzer/dist-types/models/models_0.js';
```

Comment:
Is this type not exported from the package?

Reply:
this is a mistake, will fix.